src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import {Provider} from 'react-redux';
 import App from './App';
 import './redux-js/index';
+import './components/Auth/Login/css/login.scss';
 import registerServiceWorker from './registerServiceWorker';
```

Comment:
Remove the `registerServiceWorker`, we agreed that this should not be there.

Reply:
Done